pkg/daemon/runtimes_test.go

```
@@ -0,0 +1,87 @@
+
+// TestBuildRuntimesWiresBothRuntimes guards the wiring obligation BridgeDeps
+// does not enforce by construction: daemon.go calls every setter twice, once
+// per socket-bridge runtime, and a setter called for only one of them leaves
+// that runtime's capability dead at boot with no compile-time signal (#679 —
+// SetEngine was Deno-only, so dicode.run_task failed for every Python task).
+func TestBuildRuntimesWiresBothRuntimes(t *testing.T) {
+	database, err := db.Open(db.Config{Type: "sqlite", Path: ":memory:"})
+	if err != nil {
+		t.Fatalf("open db: %v", err)
+	}
+	defer database.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `database.Close` is not checked

(errcheck)